R/plot.R

```diff
@@ -573,27 +498,66 @@ plot_predictions <- function(data = NULL,
       )
   }
 
-  plot <- plot +
-    ylab("True and predicted values")
-
-  # facet if specified by the user
-  if (!is.null(facet_formula)) {
-    if (facet_wrap_or_grid == "facet_wrap") {
-      plot <- plot +
-        facet_wrap(facet_formula, scales = scales, ncol = ncol)
-    } else {
-      plot <- plot +
-        facet_grid(facet_formula, scales = scales)
-    }
-  }
-
   return(plot)
 }
 
 
 
+#' @title Filter Data Used for Plotting
+#'
+#' @description
+#' Filter data before it gets passed to [plot_predictions()] using arbitrary
+#' filter statements. In contrast to the usual behaviour of filtering functions,
+#' rows do net get removed, but instead corresponding values of either
+#' 'true_value' or 'predction' will be replaced with `NA`. This allows to filter
```

Comment:
Prediction typo

Reply:
Re passing in separate truth and forecasts: We got rid of this for everything else as we decided it was confusing to have both options all the time. You need to have it in one data.frame anyway to score, so I think it would be most natural to keep that requirement here as well

Reply:
okedokie.